recipes/c-dbg-macro/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+from conans import ConanFile, tools
+
+
+class DbgMacroConan(ConanFile):
+    name = "c-dbg-macro"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/eerimoq/dbg-macro"
+    license = "MIT"
+    description = "A dbg(...) macro for C"
+    topics = ("conan", "debugging", "macro", "pretty-printing", "header-only")
+    settings = ("compiler", )
```

Comment:
```suggestion
    settings = ("compiler",  "build_type", "os", "arch")
```
better to specify all the settings